### PR TITLE
Update de.json

### PR DESCRIPTION
**Pull request by stephanduesterhoeft**
Added a few translations

### CODE REVIEW

**Comment by stephanduesterhoeft** [comment]
Do I have to contribute anything else? I have found other strings to translate and would like to submit them. However, these changes must first be accepted.

File: de.json [hunk 1]
```diff
@@ -980,10 +980,10 @@
 			"tooltip-clear-history": "Suchverlauf löschen",
 			"label-path-option-description": "Pfad stimmt überein",
 			"label-file-name-option-description": "Dateiname stimmt überein",
-			"label-tag-option-description": "suche nach Tags",
+			"label-tag-option-description": "Tag stimmt überein",
 			"label-line-option-description": "suche Begriff in der gleichen Zeile",
 			"label-section-option-description": "suche Begriff unter der gleichen Überschrift",
-			"label-property-option-description": "match property",
+			"label-property-option-description": "Eigenschaften stimmt überein",
```
Comment (joethei) [inline de.json:986]: "Eigenschaft stimmt überein"

Comment (stephanduesterhoeft) [reply de.json:986]: Hi @joethei, could you be a bit more specific? Do you see a mistake in the translation?

Comment (joethei) [reply de.json:986]: The English original is using singular, while your translation is using plural.
(and only partially at that)

Comment (stephanduesterhoeft) [reply de.json]: Danke. Hatte Tomaten auf den Augen. Habe es entsprechend korrigiert.

**Pull request merged**